tpot/driver.py

```diff
@@ -213,18 +213,20 @@ def main():
 
     operator_dict = None
     if args.CONFIG_FILE:
-        try:
-            with open(args.CONFIG_FILE, 'r') as input_file:
-                file_string =  input_file.read()
-            operator_dict = eval(file_string[file_string.find('{'):(file_string.rfind('}') + 1)])
-        except:
-            raise TypeError('The operator configuration file is in a bad format or not available. '
-                            'Please check the configuration file before running TPOT.')
+        if args.CONFIG_FILE == 'TPOTlight':
+            operator_dict = args.CONFIG_FILE
+        else:
+            try:
+                with open(args.CONFIG_FILE, 'r') as input_file:
+                    file_string =  input_file.read()
+                operator_dict = eval(file_string[file_string.find('{'):(file_string.rfind('}') + 1)])
+            except:
+                raise TypeError('The operator configuration file is in a bad format or not available. '
+                                'Please check the configuration file before running TPOT.')
 
     tpot = tpot_type(generations=args.GENERATIONS, population_size=args.POPULATION_SIZE,
                      offspring_size=args.OFFSPRING_SIZE, mutation_rate=args.MUTATION_RATE, crossover_rate=args.CROSSOVER_RATE,
-                     cv=args.NUM_CV_FOLDS, n_jobs=args.NUM_JOBS,
-                     scoring=args.SCORING_FN,
+                     cv=args.NUM_CV_FOLDS, n_jobs=args.NUM_JOBS, scoring=args.SCORING_FN,
```

Comment:
Do we really still call it `NUM_CV_FOLDS` in the driver? Let's fix that and just call it `CV`. I know, unrelated to this PR, but let's fix it while we notice it.